### PR TITLE
Modify container image for bots/infrastructure migration

### PR DESCRIPTION
**Pull request by nikolaszimmermann**
Contains a few individual changes, needed for the bot/infrastructure migration.

### CODE REVIEW

File: scripts/host-only/wkdev-sdk-bakery [hunk 1]
```diff
@@ -58,7 +59,16 @@ build_image() {
 
     pushd "$(get_image_directory_by_name "${container_image_name}")" &>/dev/null
     timer_start
-    run_podman_silent_unless_verbose build --jobs $(get_number_of_cores_for_build) --tag "$(get_tag_for_build)" . || _abort_ "Container image build failed"
+
+    local podman_argument=("--jobs" "$(get_number_of_cores_for_build)")
+    podman_argument+=("--tag" "$(get_tag_for_build)")
+
+    for environment_variable in "${cumulated_values_env[@]}"
+    do
```
Comment (TingPing) [inline scripts/host-only/wkdev-sdk-bakery:67]:
```suggestion
    for environment_variable in "${cumulated_values_env[@]}"; do
```

File: images/wkdev_sdk/Containerfile [hunk 2]
```diff
@@ -145,5 +141,11 @@ RUN gst-inspect-1.0 audiornnoise && \
     gst-inspect-1.0 livesync && \
     gst-inspect-1.0 rsrtp
 
+# Remove systemd services that would startup by default, when spawning
+# systemd as PID 1 within the container (usually, we don't spawn systemd
+# within the wkdev-sdk container, for interactive usage, but we do so
+# when deploying the wkdev-sdk image as bot).
+RUN find /etc/systemd/ -type l -name '*.service' -print -exec rm {} \;
```
Comment (TingPing) [inline images/wkdev_sdk/Containerfile:148]: I do worry about this causing an easy to miss issue in the future, however looking at the current services there I don't think there is a problem.

**Review by TingPing — APPROVED** [review]
LGTM

File: images/wkdev_sdk/Containerfile (hunk 2 above)
Comment (TingPing) [reply images/wkdev_sdk/Containerfile:148]: Also it shouldn't be a problem to actually remove things like cupsd (socket activated anyway) and pipewire from the image, we only use them as clients, if the packages allow that.

Use do use httpd for some testing though.

**Comment by TingPing** [comment]
CI fails with `no space left on device` (@dpino ?)

File: scripts/host-only/wkdev-sdk-bakery [hunk 3]
```diff
@@ -58,7 +59,16 @@ build_image() {
 
     pushd "$(get_image_directory_by_name "${container_image_name}")" &>/dev/null
     timer_start
-    run_podman_silent_unless_verbose build --jobs $(get_number_of_cores_for_build) --tag "$(get_tag_for_build)" . || _abort_ "Container image build failed"
+
+    local podman_argument=("--jobs" "$(get_number_of_cores_for_build)")
+    podman_argument+=("--tag" "$(get_tag_for_build)")
```
[the remark this answers is not in the document]
Comment (nikolaszimmermann) [reply scripts/host-only/wkdev-sdk-bakery:64]: No to add the -e variables below 😆

File: images/wkdev_sdk/Containerfile (hunk 2 above)
Comment (nikolaszimmermann) [reply images/wkdev_sdk/Containerfile:148]: OK, we can always refine. This is most likely not the last change of this kind.

**Comment by nikolaszimmermann** [comment]
> CI fails with `no space left on device` (@dpino ?)

A bit ironic - that is why we moved to our own runners in the first place 😊

**Comment by nikolaszimmermann** [comment]
I manually ran `podman image prune` on the host that runs the gitlab-runner. Should be fine now.

**Comment by nikolaszimmermann** [comment]
Oops, I diddn't intend to squash before merging, sorry :/

File: scripts/host-only/wkdev-sdk-bakery (hunk 1 above)
Comment (nikolaszimmermann) [reply scripts/host-only/wkdev-sdk-bakery:67]: Missed that suggestion, sorry -- will fix it up later.

**Pull request merged**